docs/UserGuide.md

```diff
@@ -188,36 +190,24 @@ Allows the user to view students and their information which includes:
 - Amount owed
 - Additional notes
 
-Format: `list`
+Format: `list`
 
 ![UiUList](images/UiList.png)
 
 ---
-### View all students: `statistics`
-
-Shows the overall statistics for the tutor, which includes:
-- Total number of students
-- Amount of money earned so far
-- Amount of money still being owed
-
-Format: `statistics`
-
-![UiStatistics](images/UiStatistics.png)
-
----
-### Exiting the program : `exit`
+### Exiting the program : `exit`
 
 Exits the program.
 
-Format: `exit`
+Format: `exit`
 
 ### Saving the data
 
 Teacher’s Pet data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-Teacher’s Pet data is saved as a TXT file `[JAR file location]/data/teacher_pet.txt`. Advanced users are welcome to edit the data file.
+Teacher’s Pet data is saved as a TXT file `[JAR file location]/data/teacher_pet.txt`. Advanced users are welcome to edit the data file.
```

Comment:
With further knowledge about the code base, it comes to my attention that AB3 is currently using a JSON file to store data instead of txt file like our IP. Therefore, I think we should stick to a JSON file instead of txt.